templates/nvidia.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1alpha1
+kind: Addon
+metadata:
+  name: nvidia
```

Comment:
Does it depends on other addons to work?

Reply:
Nope, but it depends on the konvoy cluster config to enable it:

```
  nodePools:
  - name: worker
    gpu:
      nvidia:
        enabled: false
  - name: gilbert
    gpu:
      nvidia:
        enabled: true
```

If not turned on, it is no-op - the addon will not be installed on nodes that haven't enabled Nvidia GPU.

Reply:
We might need to validate that in kubeaddons

Reply:
> We might need to validate that in kubeaddons

Kubeaddons proper treats the configuration of the Addons as opaque, so at least presently it does not have a feature for inline or automatic reconfiguration of Addons.